tests/models/maskformer/test_image_processing_maskformer.py

```diff
@@ -598,7 +600,9 @@ def test_post_process_instance_segmentation(self):
             self.assertTrue("segments_info" in el)
             self.assertEqual(type(el["segments_info"]), list)
             self.assertEqual(len(el["segmentation"].shape), 3)
-            self.assertEqual(el["segmentation"].shape[1:], (384, 384))
+            self.assertEqual(
```

Comment:
Shouldn't we also test expected slices?

Reply:
@NielsRogge I can add a test to the modeling test file since this is a dummy file but it'd great to merge this PR first.

Reply:
Ok!